covid_moonshot_ml/datasets/utils.py

```diff
@@ -483,6 +483,7 @@ def parse_fragalysis_data(frag_fn, x_dir, cmpd_ids=None, o_dir=False):
         cmpd_id = data["Compound ID"]
         dataset = data["Dataset"]
         if len(dataset) > 0:
+            ## TODO: is this the behaviour we want? this will build an empty object if there isn't a dataset
```

Comment:
not sure if this needs to be resolved in this PR, but this is a good point. might make more sense to just not add anything if the `"Dataset"` value is empty